bugsnag/src/main/java/com/bugsnag/OnSession.java

```diff
@@ -0,0 +1,5 @@
+package com.bugsnag;
+
+interface OnSession {
+    void onSession(SessionPayload payload);
```

Comment:
On other platforms `onSession` returns a `boolean` to allow sessions to be suppressed

```suggestion
    boolean onSession(SessionPayload payload);
```

Reply:
this has been implemented for session and error callbacks now